service_container/tags.rst

```diff
@@ -515,6 +515,45 @@ To answer this, change the service declaration:
             ;
         };
 
+.. tip::
+
+    In XML and YAML format, you may provide the tag an attribute called ``name``.
```

Comment:
I don't understand this "feature".

* Is the `name` attribute special here?
* If it's special: what does that imply?
* If it's not special: why mention this at all? If it's just like any other attribute, why mention this `name` attribute in particular?

Thanks!

Reply:
By default, the `name` XML attribute contains the tag name, and so cannot be used to set an attribute named `name`. As of 5.1, there is a way to use `name` as an attribute by passing the tag name in the body of the XML element instead.

Reply:
Thanks for your explanation. Now I can fully understand what's happening here. So, I've done some tweaks while merging to hopefully make it less confusing to readers: https://github.com/symfony/symfony-docs/commit/6e6565b02e09210a11f98f0adcf470c958d86f08